public/i18n/en.json

```diff
@@ -179,7 +179,7 @@
   "ftm-requesters-count": "$1 requesters",
   "ftm-about-us-title": "About us",
   "ftm-about-us-what": "What is #findthemasks?",
-  "ftm-about-us-what-details": "<strong>#findthemasks is a global mapping tool and interactive directory for PPE needs.</strong> Our mission is to enable individuals, communities, makerspace networks, and large-scale manufacturers to engage and assist during the COVID-19 pandemic by providing our frontline healthcare and other essential workers with the personal protective equipment they need to protect themselves.",
+  "ftm-about-us-what-details": "<strong> #findthemasks is a global mapping tool and interactive directory for PPE needs.</strong> Our mission is to enable individuals, communities, makerspace networks, and large-scale manufacturers to engage and assist during the COVID-19 pandemic by providing our frontline healthcare and other essential workers with the personal protective equipment they need to protect themselves.",
```

Comment:
why do these <strong> tags show up in the herokku site instead of bolding the text?

Reply:
Hahah -- when we have HTML in translation text that needs to be specified where the translation text is rendered. I'll add it

Reply:
Ohhhh is that the [html] necessary in .handlebars? Ok! I get it now - previously I just labelled it for a href= links but now I understand it’s a label for HTML language coooooool!! I’ll be more careful next time, thanks!!

Reply:
yeah basically if any strings in en.json contain any HTML tags in them, you need to add [html] before the key so that it is inserted as a string rather than text

Reply:
That makes so much sense! Ok, got it for next time